src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,37 +1,36 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
 import ColorPickerPalette from './ColorPickerPalette';
-import { AnnotationMode } from '../annotations/types';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerControl.scss';
 
+const colors = Object.values(AnnotationColor);
```

Comment:
Instead of importing the colors here and passing them in, should they pass through from `ColorPickerControl` down to `ColorPickerPalette`? So essentially it would be `(Doc|Image)Controls` that imports the colors and passes it down

Reply:
Yes I think that makes sense.